package.json

```diff
@@ -21,82 +21,80 @@
   },
   "homepage": "https://github.com/openstax/react-components",
   "dependencies": {
-    "bootstrap": "~3.3.2",
-    "camelcase": "^1.0.2",
-    "classnames": "^2.1.3",
-    "eventemitter2": "^0.4.14",
-    "font-awesome": "^4.4.0",
-    "keymaster": "^1.6.2",
-    "markdown-it": "^4.1.1",
-    "moment": "^2.9.0",
-    "underscore": "~1.8.2"
+    "bootstrap": "3.3.5",
+    "camelcase": "1.2.1",
+    "classnames": "2.1.5",
+    "eventemitter2": "0.4.14",
+    "font-awesome": "4.3.0",
```

Comment:
I was making this match what I pushed in for openstax/tutor-js#928, although we could probably safely go with the upgraded version for both repos